packages/ndla-ui/src/Spinner/Spinner.tsx

```diff
@@ -30,17 +41,8 @@ const SpinnerDiv = styled('div')`
   }
 `;
 
-const Spinner = ({ size, margin, inverted }) => <SpinnerDiv size={size} margin={margin} inverted={inverted} />;
-
-Spinner.propTypes = {
-  size: PropTypes.oneOf(['large', 'medium', 'normal']),
-  margin: PropTypes.string,
-  inverted: PropTypes.bool,
-};
-
-Spinner.defaultProps = {
-  size: 'large',
-  margin: `${spacing.normal} auto`,
-};
+const Spinner: React.FC<Props> = ({ size = 'large', margin = `${spacing.normal} auto`, inverted = false }) => (
```

Comment:
Samme.
```suggestion
const Spinner = ({ size = 'large', margin = `${spacing.normal} auto`, inverted = false }: Props) => (
```

I tillegg, dersom Spinner er gjort om fra js til ts så må eksporten flyttes fra index-javascript.js til index.ts.